hydromt/models/model_api.py

```diff
@@ -1273,6 +1281,8 @@ def write_geoms(self, fn: str = "geoms/{name}.geojson", **kwargs) -> None:
             _fn = join(self.root, fn.format(name=name))
             if not isdir(dirname(_fn)):
                 os.makedirs(dirname(_fn))
+            if kwargs.get("driver") == "GeoJSON" and to_wgs84:
```

Comment:
```suggestion
            if to_wgs84 and (kwargs.get("driver") == "GeoJSON" or str(fn).lower().endswith(".geojson")):
```

Reply:
I would be good to also remove lines 1272-73. Geopandas itself infers the driver based on the file path extension, see https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoDataFrame.to_file.html